.github/workflows/Dockerfile.dev.yml

```diff
@@ -9,6 +9,10 @@ on:
     # UTC-8: At 02:15 on every day-of-week from Monday through Friday.
     - cron: '15 10 * * 1-5'
 
```

Comment:
💭  can you add a comment saying the intention here? The github expression took me a good minute to understand what it was saying. Eg.
`# only cancel jobs on master branch if triggered by same commit`

Reply:
Good idea. Done in 7bb63d4